client/src/components/TransferFunds.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+
+import { NumberInput } from 'plaid-threads/NumberInput';
+import { Button } from 'plaid-threads/Button';
+import { currencyFilter } from '../util';
+
+interface Props {}
+const TransferFunds: React.FC<Props> = (props: Props) => {
+  const [transferAmount, setTransferAmount] = useState('');
```

Comment:
I'm wondering why we use a string here. Shouldn't transferAmount be a numeric value?

Reply:
the numberInput in Threads takes a string!!!